docs/src/user_docs/stdlib/index.md

```diff
@@ -36,6 +36,7 @@ Currently, Miden standard library contains just a few modules, which are listed
 | [std::stark::mod](./stark.md#stdstarkmod)                      | Contains procedures and helpers used when verifying STARK proofs inside the VM.                                                                                 |
 | [std::crypto::hashes::blake3](./crypto/hashes.md#blake3)            | Contains procedures for computing hashes using BLAKE3 hash function.                                                                                             |
 | [std::crypto::hashes::keccak256](./crypto/hashes.md#keccak256)      | Contains procedures for computing hashes using Keccak256 hash function.                                                                                          |
+| [std::crypto::hashes::sha2_512](./crypto/hashes.md#sha2-512)        | Contains procedures for computing hashes using SHA2‑512 hash function.                                                                                           |
 | [std::crypto::hashes::sha256](./crypto/hashes.md#sha256)            | Contains procedures for computing hashes using SHA256 hash function.                                                                                             |
```

Comment:
Should we name the module `sha512`? This would be consistent with `sha256` - or is there a reason why we want to have these named inconsistently?